docs/articletemplate.md

```diff
@@ -1,5 +1,27 @@
 ##### ARTICLE-TEMPLATE
 
+Create an article-template
+
+
```

Comment:
Maybe we should mention that we expect valid (x)html in here and that we'll tidy up the code...